api/index.ts

```diff
@@ -101,6 +103,17 @@ class ContainerAPI {
     }
     return { register, add, query, pause, resume, cancel, retry, migrate }
   }
+
+  public getNetworkQueueInstance(): INetworkQueueInstance {
```

Comment:
can return and NetworkQueue instance directly and make methods public if they need to be accessed public